src/main/webapp/data/developers.json

```diff
@@ -1583,100 +1610,85 @@
       "multiple": true,
       "name": "Zhu Liang",
       "username": "paradite"
+    },
+    {
+      "multiple": true,
+      "name": "Priyamal Madushanka",
```

Comment:
This shouldn't be at the end? It begins with P.

Reply:
Oops